catalog/files/impl/src/main/java/org/projectnessie/catalog/files/s3/S3BucketOptions.java

```diff
@@ -88,11 +94,31 @@ public interface S3BucketOptions {
    */
   Optional<String> region();
 
+  /**
+   * The authentication mode to use by the Catalog server. If not set, the default is {@code
+   * STATIC}. Depending on the authentication mode, other properties may be required.
+   *
+   * <p>Valid values are:
+   *
+   * <ul>
+   *   <li>{@code DEFAULT}: Default credentials provider chain.
```

Comment:
This feels a bit odd. Above it says, "the default is static" - but this value is named DEFAULT, which is not default.
Maybe rename `DEFAULT` to `APPLICATION_GLOBAL` or so.